src/containers/AdminDashboard/Category/RenderCategories.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import SaveIcon from '@material-ui/icons/Save';
+import Grid from '@material-ui/core/Grid';
+import { List, TextField, ListItem, ListItemText } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { categoryService } from '../../../services/category/category.service';
+import { withTranslation } from 'react-i18next';
+
+
+const styles = theme => ({
+	root: {
+		width: '100%',
+		maxWidth: 360,
+		backgroundColor: theme.palette.background.paper,
+		left: '40%',
+		position: 'absolute',
+	},
+	list: {
+		//TODO make the size responsive
+		maxHeight: 360,
+		overflow: 'scroll',
+
+	}
+});
+
+class RenderCategories extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			category: {
+				name: '',
+				description: ''
+			},
+			categories: []
+		};
+
+		this.handleOnChange = this.handleOnChange.bind(this);
+		this.submit = this.submit.bind(this);
+	}
+
+	componentDidMount() {
+		categoryService.getAll().then(data => {
+			this.setState({categories: data});
+		});
+	}
+
+	handleOnChange(event) {
+		let category = this.state.category;
+		category[event.target.name] = event.target.value;
+		this.setState({category: category});
+	}
+
+	submit() {
+		categoryService.create(
+			this.state.category).then(data => {
+			const newCategories = this.state.categories.concat(data);
+			this.setState({categories: newCategories});
+		});
+	}
+
+	render() {
+
+		const{classes, t} = this.props;
+
+		return (
+			<div>
```

Comment:
Brug fragment (<>) i stedet for div, ved mindre du har en rigtig god grund til at bruge div

Reply:
👍